Sources/Helpers/OAAppSettings.h

```diff
@@ -777,6 +777,7 @@ typedef NS_ENUM(NSInteger, EOARateUsState)
 @property (nonatomic) OACommonBoolean *depthContoursPurchased;
 @property (nonatomic) OACommonBoolean *contourLinesPurchased;
 @property (nonatomic) OACommonBoolean *wikipediaPurchased;
+@property (nonatomic) OACommonBoolean *sensorsPurchased;
```

Comment:
Plugin sensors is available with pro subscription. It cannot be purchased as InApp. So this setting and corresponding code should be removed.